draft-ietf-quic-recovery.md

```diff
@@ -1695,7 +1707,7 @@ No significant changes.
 
 - Path validation can be used as initial RTT value (#2644, #2687)
 - max_ack_delay transport parameter defaults to 0 (#2638, #2646)
-- Ack Delay only measures intentional delays induced by the implementation
+- ACK delay only measures intentional delays induced by the implementation
```

Comment:
Up above 'Delay' is capitalized, unless I'm missing something.

Reply:
There should only be "ACK Delay" for the ACK frame field and "acknowledgment delay" for the value/the concept. Let me double check this.